packages/playground/src/pages/Wallets/Wallets.js

```diff
@@ -26,13 +26,22 @@ getPaymentMethods({ amount, shopperLocale }).then(async paymentMethodsResponse =
     const amazonCheckoutSessionId = urlSearchParams.get('amazonCheckoutSessionId');
     const step = urlSearchParams.get('step');
 
+    const chargeOptions = {
+        // chargePermissionType: 'Recurring',
```

Comment:
Are we just keeping this commented out code here to show the potential shape of the object?
Can we do that with a type instead?